evansmusomi/react-native-102
# Stores Picked Image in Redux Store

*What this does*
- Stores the selected image in the store

*How to verify*
- When sharing a place, you need to pick an image in order for the share button to be enabled
- A valid image in the state is required to share a place
- Alternatively, you can verify using a react native debugger that the state is updated with an image
